.papr.yml

```diff
@@ -1,145 +1,9 @@
-# This suite skips the RPMs and does the build+unit tests in a container
+# Stub until we drop PAPR entirely
```

Comment:
Mehh... let's just nuke it if we're not gonna leave any job in it? Once we drop PAPR here, I think it's just bubblewrap really that needs to migrate off before we can kill it entirely!

Reply:
I worried that papr would barf if there wasn't a `papr.yaml` until we removed the webhooks entirely?

Reply:
It should be able to handle it gracefully.